File: Python/01 - 10/03 - eulerThree.py
def main():
    n = 600851475143
    for i in range(1, n, 1):
        if (n % i == 0):
            n = n / i
            if (n == 1):
                print("The largest prime factor of 600851475143 is", i)

if __name__ == "__main__":
    main()

File: C++/01 - 10/03_eulerThree.cpp
// What is the largest prime factor of the number 600851475143?
#include <iostream>
#include <cmath>
int main() {
    long number = 600851475143, i;
    for(i = 3; i <= number; i += 2) {
        while(number % i == 0) {
            number /= i;
        }
        if(number == 1) {
            break;
        }
    }
    std::cout << i << " is the largest prime factor of 600851475143\n";
    return 0;
}